cmd/csaf_uploader/main.go

```diff
@@ -296,6 +296,9 @@ func (p *processor) process(filename string) error {
 	if resp.StatusCode != http.StatusOK {
 		uploadErr = fmt.Errorf("upload failed: %s", resp.Status)
 		fmt.Printf("HTTPS %s\n", uploadErr)
+		if resp.StatusCode == http.StatusForbidden {
+			return err
```

Comment:
@s-l-teichmann 
We can upload several files, should be try them all, even if we are getting errors on some or abort on the first error?

Reply:
@bernhardreiter Yes, maybe we should continue with the next one.

Reply:
@s-l-teichmann right now, we do. Okay, then this remark is resolved.